dotnet/src/Experimental/Experimental.Prompty.UnitTests/Experimental.Prompty.UnitTests.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We are moving to a pattern of having unit test projects beside the code they test.

Can you move this project to `Extensions\PromptTemplates.Prompty.UnitTests`